include/rcutils/logging.h

```diff
@@ -538,18 +538,23 @@ void rcutils_logging_console_output_handler(
  * Usually it is unnecessary to call the macro directly.
  * All logging macros ensure that this has been called once.
  */
-#define RCUTILS_LOGGING_AUTOINIT \
-  if (RCUTILS_UNLIKELY(!g_rcutils_logging_initialized)) { \
-    rcutils_ret_t ret = rcutils_logging_initialize(); \
-    if (ret != RCUTILS_RET_OK) { \
-      RCUTILS_SAFE_FWRITE_TO_STDERR( \
-        "[rcutils|" __FILE__ ":" RCUTILS_STRINGIFY(__LINE__) \
-        "] error initializing logging: "); \
-      RCUTILS_SAFE_FWRITE_TO_STDERR(rcutils_get_error_string().str); \
-      RCUTILS_SAFE_FWRITE_TO_STDERR("\n"); \
-      rcutils_reset_error(); \
-    } \
-  }
+
+#define RCUTILS_LOGGING_AUTOINIT
```

Comment:
I think a better approach is to create a `define` which de/activate the logging.

Reply:
Updated to required approach. Added flag to build system here: https://github.com/micro-ROS/micro-ros-build/pull/104/commits/b6416f970f0870c2e4a6e83e947e6739f20afd7e

Reply:
What about the other logging functions?

Reply:
Please check the new changes